### PR TITLE
sigi: init at 2.1.1

### PR DESCRIPTION
**Pull request by booniepepper**
###### Motivation for this change

I've made a lot of use of this organization CLI in 2021, and others have too. I'd like to share it around.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
  - __Note: this is not applicable, nothing depends on it.__
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/sigi/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    sha256 = "sha256-y0m1AQE5qoUfPZjJfo7w5h+zZ1pbz8FkLFDM13MTWvQ=";
+  };
+
+  cargoSha256 = "sha256-NTjL57Y1Uzk5F34BW3lB3xUpD60Opt0fGWuXHQU5L3g=";
+
+  passthru.tests.version = testVersion { package = sigi; };
+
+  meta = with lib; {
+    description = "CLI tool for organization and planning";
+    homepage = "https://github.com/hiljusti/${pname}";
```
Comment (legendofmiracles) [inline pkgs/applications/misc/sigi/default.nix:20]: Substituting pname here only makes the code harder to read and unclickable from github.

File: pkgs/applications/misc/sigi/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ hiljusti ];
+    mainProgram = "sigi";
```
Comment (legendofmiracles) [inline pkgs/applications/misc/sigi/default.nix:23]: This is default.

**Review by legendofmiracles — COMMENTED** [review]
@ofborg build sigi

File: pkgs/applications/misc/sigi/default.nix (hunk 1 above)
Comment (hiljusti) [reply pkgs/applications/misc/sigi/default.nix:20]: I'll update

File: pkgs/applications/misc/sigi/default.nix (hunk 2 above)
Comment (hiljusti) [reply pkgs/applications/misc/sigi/default.nix:23]: All 4 lines? Or just the `mainProgram`?

Comment (hiljusti) [reply pkgs/applications/misc/sigi/default.nix]: I'll remove the `mainProgram` line

**Review by legendofmiracles — APPROVED** [review]
Runs and builds

**Comment by legendofmiracles** [comment]
@ofborg build sigi

Last time didn't work 😅

**Comment by hiljusti** [comment]
Thanks for looking @legendofmiracles 

The first ofborg build passed, but got invalidated when I updated with a force-push.

Looks like the build is good now, though

**Pull request merged**